Source/Events.Store.MongoDB/Processing/Streams/AbstractStreamProcessorState.cs

```diff
@@ -10,8 +10,10 @@ namespace Dolittle.Runtime.Events.Store.MongoDB.Processing.Streams
 {
     /// <summary>
     /// Represents the base state of an <see cref="AbstractScopedStreamProcessor" />.
+    /// It has a programmatically assigned <see cref="StreamProcessorStateDiscriminatorConvention"/> which takes care of
+    /// serializing <see cref="StreamProcessorState"/> and <see cref="Partitioned.PartitionedStreamProcessorState"/> to
+    /// this collection.
```

Comment:
This comment is a bit misleading - as there really is never an instance of an `AbstractScopedStreamProcessor`. Also the last three lines should probably be a remark pointing to the `StreamProcessorStateDiscriminatorConvention` which should describe what it does. So something like:
```
The <see cref="StreamProcessorStateDiscriminatorConvention"/> is used to deserialise either a <see cref="StreamProcessorState"/> or a <see cref="Partitioned.PartitionedStreamProcessorState"/> from a stream processor state collection.
```

Reply:
Fixed the comment, the `AbstractScopedStreamProcessor` seems to be a typo I did.